providers/sendinblue/src/lib/sendinblue.provider.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  ChannelTypeEnum,
+  IEmailOptions,
+  IEmailProvider,
+  ISendMessageSuccessResponse,
+} from '@notifire/core';
+import {
+  SendSmtpEmail,
+  SendSmtpEmailTo,
+  TransactionalEmailsApi,
+  TransactionalEmailsApiApiKeys,
+} from '@sendinblue/client';
+
+export class SendinblueEmailProvider implements IEmailProvider {
+  id = 'sendinblue';
+  channelType = ChannelTypeEnum.EMAIL as ChannelTypeEnum.EMAIL;
+  private transactionalEmailsApi: TransactionalEmailsApi;
+
+  constructor(
+    private config: {
+      apiKey: string;
+    }
+  ) {
+    this.transactionalEmailsApi = new TransactionalEmailsApi();
+    this.transactionalEmailsApi.setApiKey(
+      TransactionalEmailsApiApiKeys.apiKey,
+      this.config.apiKey
+    );
+  }
+
+  async sendMessage(
+    options: IEmailOptions
+  ): Promise<ISendMessageSuccessResponse> {
+    const email = new SendSmtpEmail();
+    email.sender = { email: options.from || options.from };
+    email.to = getFormattedTo(options.to);
+    email.subject = options.subject;
+    email.htmlContent = options.html;
+    email.textContent = options.text;
+    email.attachment = options.attachments?.map((attachment) => ({
+      name: attachment?.name,
+      content: attachment?.file?.toString('utf8'),
```

Comment:
@galezra Have you tried doing a test case of sending it here? I'm not sure about the utf8 string here, in the docs looks like they say it should be either a relative path or a base64 encoding. ( https://github.com/sendinblue/APIv3-nodejs-library/blob/master/docs/SendSmtpEmail.md )

If you haven't tried it our I can maybe create a free account there later to try and see if the attachments are sent and can be opened on the other side.